crates/sui-types/src/gas.rs

```diff
@@ -185,24 +208,22 @@ impl<'a> SuiGasStatus<'a> {
     }
 
     pub fn charge_min_tx_gas(&mut self) -> Result<(), ExecutionError> {
-        self.deduct_computation_cost(&INIT_SUI_COST_TABLE.min_transaction_cost)
+        self.deduct_computation_cost(INIT_SUI_COST_TABLE.min_transaction_cost.deref())
     }
 
     pub fn charge_consensus(&mut self) -> Result<(), ExecutionError> {
         self.deduct_computation_cost(&INIT_SUI_COST_TABLE.consensus_cost)
     }
 
     pub fn charge_publish_package(&mut self, size: usize) -> Result<(), ExecutionError> {
-        let computation_cost = INIT_SUI_COST_TABLE
-            .package_publish_per_byte_cost
-            .with_size(size);
+        let computation_cost =
+            NumBytes::new(size as u64).mul(INIT_SUI_COST_TABLE.package_publish_per_byte_cost.0);
```

Comment:
Should we use deref here instead of `.0`? and some other places

Reply:
Yes we can. Brb

Reply:
Done @emmazzz